File: C/menor_de_tres.c
#include <stdio.h>

int main() {
    
    int numero1, numero2, numero3, menorValor;
    
    printf("Primeiro Valor: ");
    scanf("%d", &numero1);
    
    printf("Segundo Valor: ");
    scanf("%d", &numero2);
    
    printf("Terceira Valor: ");
    scanf("%d", &numero3);
    
    menorValor = numero1;
    
    if (numero2 < menorValor) {
        menorValor = numero2;
    } 
    
    if (numero3 < menorValor) {
        menorValor = numero3;
    }
    
    printf("MENOR = %d", menorValor);
    
    
    return 0;
}

File: JAVA/menor_de_tres.java
import java.util.Locale;
import java.util.Scanner;

public class menor_de_tres {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int numero1, numero2, numero3, menorValor;
		
		System.out.print("Primeiro Valor: ");
		numero1 = sc.nextInt();
		
		System.out.print("Segundo Valor: ");
		numero2 = sc.nextInt();
		
		System.out.print("Terceiro Valor; ");
		numero3 = sc.nextInt();
		
		menorValor = numero1;
		
		if (numero2 < menorValor) {
			menorValor = numero2;
					
		} else if (numero3 < menorValor) {
			menorValor = numero3;
		}
		
		System.out.printf("MENOR = %d", menorValor);
		
		
		sc.close();
		
	}

}
